chart/values.yaml

```diff
@@ -24,6 +24,8 @@ configmap:
 
 # kube-httpcache specific configuration
 cache:
+  # name of backend service
+  # frontendService: kube-httpcache-headless
```

Comment:
Just a quick thought: When supplying your own frontend service, is the service created by the chart itself still needed? OTOH, it probably wouldn't do any harm to just have it created anyway.

Open to suggestions. :shrug:

Reply:
I'm still using the service created by the chart.

I'm following these approaches to get working readiness probes

https://github.com/mittwald/kube-httpcache/issues/36#issuecomment-713242670
https://github.com/mittwald/kube-httpcache/issues/36#issuecomment-740278034

Reply:
OK, I see. Nevermind, then. 😉